### DISCUSSION TITLE
[General] 기본 환경 설정 (eslint, prettier, vscode) / 코딩 컨벤션

### DISCUSSION THREAD

**Discussion by hochan222** [transcendence42/javascript-archive#3] [General]
### 기본 환경 설정 
- eslint
- prettier
- vscode
  - settings
  - extension

### 코딩 컨벤션
- https://google.github.io/styleguide/jsguide.html
- https://ui.toast.com/fe-guide/ko_CODING-CONVENTION

**Comment by hochan222** [comment]
공유하고싶은 자료 & 개인적으로 정리한 블로그 등이 있다면 여기👇👇👇에 댓글로 공유해주세요 😆

**Reply by hochan222** [reply]
- eslint rules
  - https://eslint.org/docs/rules/
  - [개인 블로그 포스팅](https://egas.tistory.com/13)
- cypress setting
  - https://docs.cypress.io/guides/overview/why-cypress#Features

**Reply by yechoi42** [reply]
- cypress 사용법
  - https://velog.io/@_woogie/E2E%ED%85%8C%EC%8A%A4%ED%8A%B8-with-Cypress

**Reply by yechoi42** [reply]
- cypress 테스트 코드 작성하기
  - https://nukeguys.github.io/dev/cypress/

**Reply by yechoi42** [reply]
- cypress TS 설정하기
  - https://docs.cypress.io/guides/tooling/typescript-support#Configure-tsconfig-json

**Reply by yechoi42** [reply]
- vscode에서 TS 설정하기
  - https://poiemaweb.com/typescript-vscode

**Reply by yechoi42** [reply]
- eslint에서 cy를 에러로 여길 때 
  - https://stackoverflow.com/questions/58982852/eslint-cy-is-not-defined-cypress
